Samples/Docker-Simple/Silo/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Microsoft.Extensions.Logging;
+using Orleans.Hosting;
+using Orleans.Runtime.Configuration;
+using System;
+using System.IO;
+using System.Net;
+using System.Runtime.Loader;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Silo
+{
+    class Program
+    {
+        private static ISiloHost silo;
+        private static readonly AutoResetEvent closing = new AutoResetEvent(false);
```

Comment:
Since you are waiting on several places for this to be signaled, you probably want to use a `ManualResetEvent` instead of Auto...
Also, probably a more intuitive name would be `siloStopped`, as this is signaled after the silo is stopped. Whether as closing is ambiguous: is when the silo is closing, when the program is closing, etc